bindata/network/ovn-kubernetes/common/ipsec-host.yaml

```diff
@@ -236,6 +236,13 @@ spec:
           ip x s flush
           ip x p flush
 
+          # Make libreswan to narrow down the proposals exchanged by two peers to a mutually acceptable set.
+          # The /etc/ipsec.d/openshift.conf file can not be used because it is managed by openvswitch.
+          # Hence this is being set as default conn parameter in /etc/ipsec.conf file.
+          if ! grep -q "narrowing=yes" /etc/ipsec.conf; then
+            echo -e "\nconn %default\n    narrowing=yes" >> /etc/ipsec.conf
```

Comment:
I wonder if it would be best to use our own file, like `/etc/ipsec.d/cno.conf`?
Also, including the comment in the config file itself might be useful.

Reply:
yes, using our own file is a good idea. changed it to use `/etc/ipsec.d/cno.conf` file and included comments.